infra/main.bicep

```diff
@@ -0,0 +1,314 @@
+//   Microsoft.Web/sites for appservice, function
+// Example usage:
+//   tags: union(tags, { 'azd-service-name': apiServiceName })
+#disable-next-line no-unused-vars
+var quarkusServiceName = 'quarkus'
+
+#disable-next-line no-unused-vars
+var springBootServiceName = 'spring-boot'
+
+/******************************************************************************/
+/*                                 RESOURCES                                  */
+/******************************************************************************/
+
+// Organize resources in a resource group
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: !empty(resourceGroupName) ? resourceGroupName : '${abbreviations.resourcesResourceGroups}${environmentName}'
+  location: location
+  tags: tags
+}
+
+module quarkus './app/quarkus.bicep' = {
+  name: 'quarkus'
+  scope: resourceGroup
+  params: {
+    name: _quarkusContainerAppName
+    location: location
+    tags: tags
+    identityName: _quarkusContainerAppName
+    applicationInsightsName: monitoring.outputs.applicationInsightsName
+    containerAppsEnvironmentName: containerAppsEnvironment.outputs.name
+    containerRegistryName: containerRegistry.outputs.name
+    databaseConfig: {
+      name: 'quarkusdb'
+      hostname: postgresServer.outputs.POSTGRES_DOMAIN_NAME
+      username: postgresAdminUsername
+    }
+    keyVaultName: keyVault.name
+    exists: quarkusAppExists
+  }
+}
+
+module springBoot './app/spring-boot.bicep' = {
+  name: 'spring-boot'
+  scope: resourceGroup
+  params: {
+    name: _springBootContainerAppName
+    location: location
+    tags: tags
+    identityName: _springBootContainerAppName
+    applicationInsightsName: monitoring.outputs.applicationInsightsName
+    containerAppsEnvironmentName: containerAppsEnvironment.outputs.name
+    containerRegistryName: containerRegistry.outputs.name
+    databaseConfig: {
+      name: 'springbootdb'
+      hostname: postgresServer.outputs.POSTGRES_DOMAIN_NAME
+      username: postgresAdminUsername
+    }
+    keyVaultName: keyVault.name
+    superHeroUrl: quarkus.outputs.SERVICE_QUARKUS_URI
+    exists: springBootAppExists
+  }
+}
+
+module containerAppsEnvironment 'core/host/container-apps-environment.bicep' = {
+  name: _containerAppsEnvironmentName
+
+  scope: resourceGroup
+  params: {
+    name: _containerAppsEnvironmentName
+    location: location
+    tags: tags
+    logAnalyticsWorkspaceName: monitoring.outputs.logAnalyticsWorkspaceName
+    applicationInsightsName: monitoring.outputs.applicationInsightsName
+  }
+}
+
+module keyVault './core/security/keyvault.bicep' = {
+  name: _keyVaultName
+  scope: resourceGroup
+  params: {
+    name: _keyVaultName
+    location: location
+    tags: tags
+    principalId: principalId
+  }
+}
+
+/* Give access to the keyvault to the current identity */
+module principalKeyVaultAccess './core/security/keyvault-access.bicep' = {
```

Comment:
See my comment in chat re moving to RBAC

Reply:
https://github.com/pamelafox/django-quiz-app/pull/53/files#diff-79375a10e2976aa15cbd746f969e2d0b33854a2a39f433f0bed42b66dbd6eeee